lib/routemaster/cache.rb

```diff
@@ -47,35 +47,33 @@ def initialize(&block)
       # @!attribute status
       # @return [Integer]
       # Delegated to the `block`'s return value.
-      
+
       # @!attribute headers
       # @return [Hash]
       # Delegated to the `block`'s return value.
-      
+
       # @!attribute body
       # @return pHashie::Mash]
       # Delegated to the `block`'s return value.
-      
+
       delegate :value => :@future
       delegate %i(status headers body) => :value
     end
 
     class Response < Hashie::Mash
       # @!attribute status
       # Integer
-      
+
       # @!attribute headers
       # Hash
-      
+
       # @!attribute body
       # Hashie::Mash
     end
 
-
     def initialize(redis:nil, fetcher:nil)
       @redis   = redis || Config.cache_redis
-      @expiry  = Config.cache_expiry
-      @fetcher = fetcher || Fetcher
+      @fetcher = fetcher || Fetcher.configure {}
```

Comment:
I know you're a fan of this DSL-style pattern. Personally, I don't see what it brings over just passing arguments to a constructor (and it's definitely harder to read as the standard Ruby way); but I won't fight as long as nothing leaks.

In this case, this means the `use` method should only be available during construction.